openkfw/Oscar
# Fix timeline

## Description


Fix timeline

## How has this been tested?

- Run ./start.sh, ./runinitialload
- Check Daily Covid-19 cases per region layer timeline, if it looks correct

## Types of changes



- [ X] Bug fix (non-breaking change which fixes an issue)
